openmls/src/group/mls_group/mod.rs

```diff
@@ -447,6 +445,12 @@ impl MlsGroup {
     pub(crate) fn proposal_store_mut(&mut self) -> &mut ProposalStore {
         self.group.proposal_store_mut()
     }
+
+    /// Resets the AAD.
+    #[inline]
+    pub(crate) fn reset_aad(&mut self) {
+        self.aad = Vec::new();
```

Comment:
You could also clear here.
```suggestion
        self.aad.clear();
```